packages/pluggableWidgets/datagrid-web/src/components/Table.tsx

```diff
@@ -183,7 +192,7 @@ export function Table<T>(props: TableProps<T>): ReactElement {
                 canPreviousPage={canPreviousPage}
                 gotoPage={gotoPage}
                 nextPage={nextPage}
-                numberOfItems={props.data.length}
+                numberOfItems={rows.length}
```

Comment:
Looks like a drastic change, was this intentional ?

Reply:
This fixes the second problem related to this PR (filtered values are not showing correct numbers in pagination), and this is the pagination when not using backend source (props.data)